trompace/queries/templates.py

```diff
@@ -20,16 +20,30 @@
 language
 }}'''
 
+QUERY_ALL_TEMPLATE = '''{queryname}{{
+identifier
```

Comment:
I don't think all of the items that we're going to query have these items to return, right? I'd be better to pass in a list of fields to the format_query method

Reply:
Done, as default, the list of items to return is just identifier and name.